lib/cunit/cu_tester.c

```diff
@@ -72,7 +72,7 @@ int main(int argc, char *argv[])
 	/* Set up to use the system memory management / logging */
 	pc_install_default_handlers();
 
-	pc_set_handlers(0, 0, 0, cu_error_reporter, 0, 0);
+	pc_set_handlers(0, 0, 0, cu_error_reporter, 0, cu_error_reporter);
```

Comment:
That was a change by @achidlow that sets the warn_handler. He may have needed it when he was trying to do his python wrapping. I will remove that change.